en/middleware/uxrce_dds.md

```diff
@@ -445,6 +445,20 @@ Each (`topic`,`type`) pairs defines:
 You can arbitrarily change the configuration.
 For example, you could use different default namespaces or use a custom package to store the message definitions.
 
+For publishers, an optional key `interval_us` can be added to specify the minimum duration between topic publications.
+If this key is not present, the topic will be published at the uORB topic rate.
+For subscribers, this key will be ignored.
+
+```yaml
+publications:
+  - topic: /fmu/out/vehicle_attitude
+    type: px4_msgs::msg::VehicleAttitude
+    interval_us: 100000
```

Comment:
So this is 10 Hz. The UORB publish rate is nominally 400Hz (say). So you're saying that this will make the attitude publish much slower?